app/http.php

```diff
@@ -1199,17 +1238,23 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
     $activeRuntimes = $register->get('activeRuntimes');
     $statsHost = $register->get('statsHost');
 
+    $networks = explode(',', str_replace(' ', '', Http::getEnv('OPR_EXECUTOR_NETWORK') ?: 'executor_runtimes'));
     /**
-     * Remove residual runtimes
+     * Remove residual runtimes and networks
      */
-    Console::info('Removing orphan runtimes...');
-
-    removeAllRuntimes($activeRuntimes, $orchestration);
-
-    Console::success("Orphan runtimes removal finished.");
+    Console::info('Removing orphan runtimes and networks...');
+    cleanUp($orchestration, $activeRuntimes, $networks);
+    Console::success("Orphan runtimes and networks removal finished.");
 
     // TODO: Remove all /tmp folders starting with System::hostname() -
 
+    /**
+     * Create and store Docker Bridge networks used for communication between executor and runtimes
+     */
+    Console::info('Creating networks...');
+    $createdNetworks = createNetworks($orchestration, $networks);
+    Http::setResource('networks', $createdNetworks);
```

Comment:
Should be a function returning $createdNetworks

Reply:
fixed thanks 👍